articles/explainable-ai/index.md

```diff
@@ -0,0 +1,164 @@
+
+#### Post-Hoc Methods
+These techniques only involve explainability during testing stages. The training stages are carried out normally. Post-hoc model analysis is a very common approach towards explaining AI in production.
+
+##### Local interpretable Model-Agnostic Explanations (LIME)
+[LIME](https://www.oreilly.com/content/introduction-to-local-interpretable-model-agnostic-explanations-lime/) is a popular post-hoc method that learns an interpretable model and attempts to explain its prediction. Only after a decision has been made will it provide a explanation. Here is how it works. LIME receives input, then generates a new dataset composed of refined data samples. The next step involves populating corresponding predictions that would have been made by a black-box model if the aforementioned samples would have been used as input.
+
+Next is the training of an interpretable model (such as regression models or decision trees). The model is trained on the new dataset to help explain changes in the key extracted features.
+
+Let’s go over an example to make it a bit clearer. Consider an image classification problem. The input is image data to be classified. Let’s take an image of a cat. LIME blacks out different parts of the input image of a cat. It feeds the resulting images through the interpretable model. This is to test the model on which perturbations on the images throw off the model's ability to predict that the image is of a cat.
+
+LIME uses this to predict the reasoning behind the decision making of the model. Taking our cat image example, LIME would find that erasing parts of the cat's face makes it more difficult for the model to identify that it is a cat. As a result, this would show that classification of an image of a cat would highly depend on the face of the cat.
+
+##### Black Box Explanation through Transparent Approximation (BETA)
+BETA is a post hoc method that is linked to Interpretable Decision Sets. [Interpretable Decision Sets](https://cs.stanford.edu/people/jure/pubs/interpretable-kdd16.pdf) is a framework used to build highly accurate and interpretive predictive models. These are sets of “if-then” rules.
+
+BETA learns decision sets through which each rule explains part of the behavior of the model. The learning process yields a high degree of agreement between the model and the explanation. The process also yields low ambiguity and high interpretability. For a more technical explanation on BETA check out this [paper](https://arxiv.org/abs/1707.01154).
```

Comment:
I did not understand the statement: ' The learning process yields a high degree of agreement between the model and the explanation' 

Can you include an example here? What are interpretable decision sets? What is the need for using them and what advantages do they offer?